### ISSUE TITLE
App forget about login and won't start

### ISSUE THREAD

**Issue by playforvoices1** [KevinCFechtel/FluxNews#29]
I use FluxNews version 1.3.6. on Sony Stock Android 12 and Huawei EMUI 13.x. My Minuflux server has version 2.1.1. 

The app can login at the first time and syncing my subscriptions. But if I work a little with the app, I get the error message that I'm not logged in. If I restart the app, she stuck at the boot screen. I have to clean the cache and all local data. Then the app starts, and I'm able to log in with my credentials and the circle starts again. 

This issue happens on both devices.

**Comment by KevinCFechtel** [comment]
Hi, could you please put the app in debug mode and export the logs when you get the message that you are not logged in?
Please do not restart the app, otherwise the logs will be overwritten.
I'll take a look at it then.

**Comment by playforvoices1** [comment]
[22032024_143725_PM_all_[1].zip](https://github.com/KevinCFechtel/FluxNews/files/14723298/22032024_143725_PM_all_.1.zip)

Here comes the Debug Log.

**Comment by KevinCFechtel** [comment]
I don't see a direct error in the logs, but I suspect that the problem lies in the storage of the API key in Android's secure storage.
This storage is realized via a plugin, I will see if I can get any information about errors.
Is the Api Key field in the settings empty after the message with the missing login data appeared?

**Comment by playforvoices1** [comment]
Yes, the API key field is empty after I get the login error message. I try a new and clean install, then I log in with my credentials and make a refresh of the feeds. This works fine. Then I go back to settings and activated the function "show feed icons"  and imminently the API key in the setting was lost and showing an empty field.

**Comment by KevinCFechtel** [comment]
Thanks for the info, that seems logical, since every change to the settings calls up a new read of the persistently saved settings. And this also seems to be the problem, these settings seem to be cleared on your devices. 
Unfortunately, I cannot reproduce the behavior on the emulator under Android 12. 
Are you familiar with reading logs via Logact? If not, I would try to write a guide, because this might be the only way to recognize the problem directly ob yout device.

**Comment by playforvoices1** [comment]
Then I need a Logcat guide, :) I wonder that the problem appears on every device I use. All on Android 13 (LineageOS and Stock ROM base).

**Comment by KevinCFechtel** [comment]
Have you installed android studio?
If so, please open it and connect your phone to the PC, then you should be able to use logcat to read the logs as described here: [working with logcat in android studio](https://developer.android.com/studio/debug/logcat)
If the logs are read out, please start the app and reproduce the error
If not, then try installing only the shell version of logcat with the following instructions and connect your phone to it:
[install adb in windows, macos or linux](https://www.xda-developers.com/install-adb-windows-macos-linux/)
If the general connection works (an `adb device` outputs your cell phone) then you can connect logcat to your cell phone with the following command:
`adb -s "device id from adb devices" logcat`
Afterwards, please start the app on your phone and try to reproduce the error once. 
You can then attach the log file here and I would take a look at it.
